### ISSUE TITLE
Wrong Block-Ref?

### ISSUE THREAD

**Issue by mschiff** [sethyuan/logseq-plugin-reminder#10]
In the notification dialog within Logseq, the Button "Show ... block" (or similar) does not take me to the correct block but instead to some random block that has no /schedule or /deadline at all, so I assume there issomething wrong with the blockref being used.

**Comment by sethyuan** [comment]
I'll take a look of it.

**Comment by sethyuan** [comment]
@mschiff I ran some quick tests and was unable to reproduce it. Can you reproduce it?

**Comment by mschiff** [comment]
@sethyuan I could reproduce it for specific events and saw later that it works as expected for other events.

The only difference I can see: When it does not work its with blocks on a page (not journal) and the schedule is in a nested block like so:

```
- # Daily
    - ## Morning
      SCHEDULED: <2023-08-06 Sun 09:00 .+1d>
        - Do this and that
    - ## Evening
      SCHEDULED: <2023-05-04 Thu 18:00 .+1d>
        - Do bar and baz
```
When it was working properly, this was a schedule in a block on the daily journal, not nested.

**Comment by sethyuan** [comment]
OK, I'll run some tests with the example you provided in a non-journal page, see if I can pinpoint the issue.

**Comment by sethyuan** [comment]
@mschiff I recreated the examples you've given above in a non-journal page, but still unable to reproduce it. What do you see when you click the button? Where does it take you?

**Comment by mschiff** [comment]
@sethyuan It takes me to a block on a page that is totally unrelated and does not even have a schedule or deadline. What other information can I give you?

**Comment by sethyuan** [comment]
What do you see on the system notification? Is the content in it correct?

**Comment by mschiff** [comment]
Yes that was correct and as expected. Today clicking on the button in the plugins dialog led me to the correct block. So it is not 100% reproducable. I will write again as soon as it happens again. Maybe (not sure anymore ;-)) I did a rebuild of the graph index and that fixed it somehow?!

**Comment by sethyuan** [comment]
OK. yeah, I suspect a re-index would correct uuid problems.
